src/unix/stdpaths.cpp

```diff
@@ -31,12 +31,22 @@
 
 #include "wx/filename.h"
 #include "wx/log.h"
+#include "wx/scopeguard.h"
 #include "wx/textfile.h"
 
 #if defined( __LINUX__ ) || defined( __VMS )
     #include <unistd.h>
 #endif
 
+#if defined(__FreeBSD__)
```

Comment:
I think this should be

```suggestion
#ifdef wxHAS_PROCSTAT
```